packages/circuit-ui/package.json

```diff
@@ -87,7 +87,7 @@
     "@emotion/styled": "11.x",
     "@sumup/collector": ">=1.4.0",
     "@sumup/design-tokens": ">=4.0.0",
-    "@sumup/icons": ">=2.8.0",
+    "@sumup/icons": ">=2.9.0",
```

Comment:
@amelako We treat patch and minor bumps of peer dependencies as a patch change on the parent package, so Circuit UI should have been bumped to 5.1.1.